CONTRIBUTING.md

```diff
@@ -151,21 +156,21 @@ Please enable sandboxing **before** building the package by adding the following
 #### Built on platform(s)
 
 Many Nix packages are designed to run on multiple platforms.
-As such, it’s important to let the maintainer know which platforms your changes have been tested on.
-It’s not always practical to test a change on all platforms, and is not required for a pull request to be merged.
-Only check the systems you tested the build on in this section.
+It’s important to let the maintainer know which platforms you have tested on.
```

Comment:
```suggestion
As such, it’s important to let the maintainer know which platforms you have tested on.
```

While i will champion active direct language any day, some mental breaks and connective tissue is needed for readability